apps/demos/Demos/Charts/PyramidChart/description.md

```diff
@@ -1,2 +1,15 @@
-The DevExtreme JavaScript Pyramid Chart is&nbsp;often used to&nbsp;visualize an&nbsp;organizational structure. In&nbsp;this demo, the pyramid displays a&nbsp;team&rsquo;s composition, reflecting both subordination and strength. In&nbsp;code, the pyramid is&nbsp;created using the Funnel component with the **algorithm** property set to _&laquo;dynamicHeight&raquo;_ and **inverted** property set to&nbsp;**true**.
-<!--split-->
\ No newline at end of file
+DevExtreme Funnel can display data in a pyramid chart. To configure the component as a pyramid chart, assign *"dynamicHeight"* to the [algorithm](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#algorithm) property and enable [inverted](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#inverted).
```

Comment:
DevExtreme Funnel can display data as a pyramid chart. To create such visualization, assign *"dynamicHeight"* value to the [algorithm](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#algorithm) property and [invert the Funnel](/Documentation/ApiReference/UI_Components/dxFunnel/Configuration/#inverted).